### DISCUSSION TITLE
What args are available to skip certain checks?

### DISCUSSION THREAD

**Discussion by guicane** [fabasoad/pre-commit-snyk#48] [General]
The readme doesn't list the available arguments to skip certain checks that are picked up. Is there a list somewhere? The example only shows one specific example which doesn't apply to what I want.

**Comment by fabasoad** [comment]
Hi @guicane! What do you mean by "skip certain checks"? Do you mean to [ignore some issue by ID](https://docs.snyk.io/snyk-cli/commands/ignore)? Or something else? Please provide an example.

**Comment by fabasoad** [comment]
Closing as stale.
